ffi/src/logging.rs

```diff
@@ -9,57 +9,58 @@ use crate::BorrowedBytes;
 pub struct LogArgs<'a> {
     /// The file path where logs for this process are stored.
     ///
-    /// If empty, this is set to `${TMPDIR}/firewood-log.txt`.
+    /// This is required and must not be empty. Use "/dev/stdout" for stdout logging.
```

Comment:
We shouldn't recommend use of `/dev/stdout` as it will cause interleaving of the stdout from the parent process -- even if the parent process is also implemented Rust sharing the same stdlib statics.

- Any synchronization in Go around printing to stdout will be bypassed.
- Because `File::open("/dev/stdout")` returns a new fd, any synchronization in Rust will also be bypassed.